src/core/types/addons.js

```diff
@@ -54,6 +55,24 @@ type EventDataType = {|
   conversion: string,
 |};
 
+export type ExternalPreviewType = {|
+  caption: LocalizedString | null,
+  image_size: [number, number],
+  image_url: string,
+  thumbnail_size: [number, number],
+  thumbnail_url: string,
+|};
+
+export type PreviewType = {|
+  h: number,
+  src: string,
+  thumbnail_h: number,
+  thumbnail_src: string,
+  thumbnail_w: number,
+  title: string,
+  w: number,
```

Comment:
If this is our internal type, can we have a better prop name than a single char?

Reply:
Yes, but before I change it I want to confirm whether you are okay with keeping these changes in this patch, or if we should leave them to a follow-up patch.

Reply:
Ah, I believe we're using `w` and `h` here because those are the property names that the `PhotoSwipeGallery` component expects, so we either need to keep them that way, or introduce another transformer to convert from `PreviewType` to an object that works with `PhotoSwipeGallery`. The latter seems totally unneccesary. How about I add a comment to the type to explain that the prop names are those needed for the `PhotoSwipeGallery` component?

Reply:
You're right. I forgot about that (although I thought about it after having left the comment). Yes, a comment sounds good, thanks